packages/contracts/contracts/L1/rollup/CanonicalTransactionChain.sol

```diff
@@ -258,13 +322,13 @@ contract CanonicalTransactionChain is ICanonicalTransactionChain, Lib_AddressRes
 
         // Transactions submitted to the queue lack a method for paying gas fees to the Sequencer.
         // So we need to prevent spam attacks by ensuring that the cost of enqueueing a transaction
-        // from L1 to L2 is not underpriced. Therefore, we define 'ENQUEUE_L2_GAS_PREPAID' as a
+        // from L1 to L2 is not underpriced. Therefore, we define 'enqueueL2GasPrepaid' as a
         // threshold. If the _gasLimit for the enqueued transaction is above this threshold, then we
         // 'charge' to user by burning additional L1 gas. Since gas is cheaper on L2 than L1, we
         // only need to burn a fraction of the provided L1 gas, which is determined by the
-        // L2_GAS_DISCOUNT_DIVISOR.
-        if(_gasLimit > ENQUEUE_L2_GAS_PREPAID) {
-            uint256 gasToConsume = (_gasLimit - ENQUEUE_L2_GAS_PREPAID) / L2_GAS_DISCOUNT_DIVISOR;
+        // l2GasDiscountDivisor.
+        if(_gasLimit > enqueueL2GasPrepaid) {
+            uint256 gasToConsume = (_gasLimit - enqueueL2GasPrepaid) / l2GasDiscountDivisor;
```

Comment:
No chance for overflow here because we did the check `_gasLimit > enqueueL2GasPrepaid` right above. Also no chance for weird edge case behavior where a super low `_gasLimit` is supplied because of `require(_gasLimit >= MIN_ROLLUP_TX_GAS)` above

Reply:
Also it'd just revert because solidity 0.8 :)